src/components/10-atoms/text/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 7.0.0
+
+Breaking change:
+- Remove variant `semibold`, because this option is not available in the styleguide
+- Remove variant `bold` use new boolean flag `bold`, because bold for all text sizes possible
```

Comment:
Remove variant `bold`; instead use new boolean flag `bold`, because bold  possible for all text sizes